src/utils/getCheckNumber.ts

```diff
@@ -100,7 +100,7 @@ export function splitNumberOperator(keyword: string): {
   }
 
   if (secondValue) {
-    if (secondValue < firstValue) {
+    if (Number(secondValue) < Number(firstValue)) {
```

Comment:
@targos Do you see a better way or a problem with this approach ?

Reply:
Seems reasonable to me